### ISSUE TITLE
Support projet local paths for renderScriptPath

### ISSUE THREAD

**Issue by EdwardSalter** [timo-reymann/intellij-mjml-support#223]
### Description

I have a custom renderer script implemented in my repository to allow for supporting templating in my MJML files.
When I set this renderer through the settings UI, the `.idea\mjmlSettings.xml` file gets created with the `renderScriptPath` property set to the full file path of my custom renderer (e.g. `<option name="renderScriptPath" value="W:\MyRepo\scripts\intellij_mjml_renderer.js" />`).

Many other xml files in the .idea directory support the use of the `$PROJECT_DIR$` replacement variable that can be used instead of a full file path (e.g. `<option name="renderScriptPath" value="$PROJCET_DIR$\scripts\intellij_mjml_renderer.js" />`). If this plugin supported that same replacement, the `mjmlSettings.xml` file could be checked in to source control so that the plugin can be automatically configured when sharing between different machines/developers where repositories will not always have the same paths.

**Comment by timo-reymann** [comment]
I took a look at this, and it should be working out of the box, as the serialize/deserialize logic from JetBrains replaces the [path variables](https://www.jetbrains.com/help/idea/absolute-path-variables.html) in saved configuration files accordingly.

After digging a bit in JetBrains forums, I found [this](https://intellij-support.jetbrains.com/hc/en-us/community/posts/207470105/comments/207242745) from 2016:

> I didn't notice that IDEA collapse paths on save automatically.
>
> My fault was to use system dependent separator (in my case '\')
Because collapse & expand only works with '/' but not with '\\'.

I tested it on my Ubuntu machines & Macbook, and it works fine on all of them. 

So I assume it's the same issue here. Currently, I set the path as is, which apparently seems to be an issue on Windows systems.

**Comment by EdwardSalter** [comment]
Thank you for looking at this so quickly!

I did try to manually modify the settings file to include some path variables before I submitted the issue but I was trying with the backslash paths.
I've just tried it again now using forward slash paths and found that it is indeed correctly replacing the values meaning that I can commmit my settings file to the repository.

**Comment by timo-reymann** [comment]
Awesome, will merge that one to main.

I would just kindly ask you to install the snapshot (ci will post the link) and try if the fix works well on Windows. I dont really have a Windows dev machine I could test it on 😅

**Comment by timo-reymann** [comment]
Comment did not work due to autoclose by referencing in MR 😅 

You can download it here:
https://plugins.jetbrains.com/plugin/download?rel=true&updateId=636483

**Comment by EdwardSalter** [comment]
Thanks for that. I was hunting around for it and couln't work out where it was 🤣

I have downloaded the snapshot and given it a go and after choosing a script within the project, it is now automatically saving to the XML file with the $PROJECT_DIR$ variable already in place and the forward slashes instead of backslashes.

The only thing I'd note is that when you have chosen the folder within the textbox, the backslashes are shown. Once you hit Apply, it updates the textbox to show forward slashes instead. It's not really an issue since it doesn't affect the functionality.

Anyway, I want to thank you for a great plugin and especially this feature which takes it beyond any of the similar plugins in VSCode I tested before finding this one.
